src/main/webapp/user_profile_style.css

```diff
@@ -0,0 +1,158 @@
+#nav-bar {
+	margin-left: 2%;
+	width: 96%;
+}
+#return-link {
+	float: left;
+}
+
+#return-link a {
+	font-family: Poppins, sans-serif;
+	font-weight: 700;
+	color: #000;
+}
+
+#log-out-link {
+	font-family: 'DM Sans', sans-serif;
+	font-weight: 600;
+	float: right;
+}
+
+#log-out-link a {
+	font-family: 'DM Sans', sans-serif;
+	font-weight: 600;
+	color: #cdcdcd;
+}
+
+#log-out-link a:hover {
+	color: blue;
+}
+
+#header {
+	margin-top: 2px;
+	margin-left: 2%;
+	margin-right: 2%;
+}
+
+#title {
+	font-size: 40px;
+	font-family: “Book Antiqua”, Palatino, “Palatino Linotype”, “Palatino LT STD”, Georgia, serif;
+	float: left;
+}
+
+#points {
+	float: right;
+	font-family: Cambria, Georgia, serif;
+}
+
+#button-container-wrap {
+	margin-top: 20px;
+	margin-left: 2%;
+}
+
+#container {
+	display: inline;
+}
+
+#container .help-button {
```

Comment:
Some of the styles here will be common across all pages, it would be worth thinking about creating a file for that and sharing

Reply:
After Mafe and Denzil commit their pages, we would set a meeting to discuss which style should be shared across all the pages. We would try to make the style for all pages nearly the same.